x-pack/plugins/canvas/canvas_plugin_src/functions/common/ply.ts

```diff
@@ -54,7 +55,7 @@ export function ply(): ContextFunction<'ply', Datatable, Arguments, Return> {
           const column = context.columns.find(col => col.name === by);
 
           if (!column) {
-            throw new Error(`Column not found: '${by}'`);
+            throw errors.columnNotFound(by);
```

Comment:
Kind of a nit, but translating field names seems wrong. The source data has an explicit string, and it seems like we should show it directly instead of this:

![Screenshot 2019-05-31 14 26 13](https://user-images.githubusercontent.com/404731/58735728-3220bd80-83b0-11e9-961c-f8f0c005d4ec.png)

Reply:
Hrm, looking at the `columnNotFound` function, it does track that string as a value, maybe that's just a side-effect of the `en-xa` locale?